kotlin/src/com/google/idea/blaze/kotlin/sync/AlwaysPresentKotlinSyncPlugin.java

```diff
@@ -0,0 +1,35 @@
+package com.google.idea.blaze.kotlin.sync;
+
+import com.google.idea.blaze.base.model.primitives.LanguageClass;
+import com.google.idea.blaze.base.plugin.PluginUtils;
+import com.google.idea.blaze.base.projectview.ProjectViewSet;
+import com.google.idea.blaze.base.scope.BlazeContext;
+import com.google.idea.blaze.base.scope.output.IssueOutput;
+import com.google.idea.blaze.base.sync.projectview.WorkspaceLanguageSettings;
+import com.intellij.openapi.project.Project;
+import org.jetbrains.annotations.Nullable;
+
+import static com.google.idea.blaze.kotlin.BlazeKotlin.PLUGIN_ID;
```

Comment:
This `PLUGIN_ID` comes from the Kotlin plugin itself. We can't have this in the always present sync plugin. Since the purpose of this is to handle the case where the kotlin plugin is absent.
I would recommend getting rid of `BlazeKotlinBaseSyncPlugin` entirely, since there are many other things which implicitly require the Kotlin plugin to be present.